### DISCUSSION TITLE
Provide a way to customize the Editor UI Colors

### DISCUSSION THREAD

**Discussion by mcrowell** [elementor/elementor#21826] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The new UI colors in Elementor 3.12 are hard on the eyes. 

### Describe the solution you'd like

Give us the option to create/change our own custom colors for the new UI.

**Comment by Joliamendi** [comment]
In my opinion, the main color of any page builder is actually a brand color for it. So I don't think it would be interesting for each user to change the element to their desired color. This is my personal opinion.
Instead, I think it's better for Elementor developers to consult with users a bit about this.

**Reply by fabigrafo** [reply]
Yeah but it's not the Elementor brand color by the way.. It's the soft pink and not the strong pink.

**Comment by shred787** [comment]
I agree. The new changes are absolutely awful. 

The actual "dark mode" settings also broke several aspects of my website by conflicting with some very well respected addons such as Engagebay.

So I have no choice but to use light-mode.

And the new light-mode setting is horrible on a large-screen monitor,

**Comment by alloutsidethebox** [comment]
Yup this color scheme is horrid. 
Perhaps when making drastic ux changes try giving us people that pay your bills a vote eh.

**Comment by jerome-hubert** [comment]
Personally, I don't like these colors and this is quite disturbing in my workflow. But my personal preferences are probably not interesting Elementor. 
What bothers me the most is that I have made hundreds of tutorial videos to train my clients and these will no longer be valid...
Clients already called me and say "_there's an error in Elementor, everything has changed_". (Well, It's just colors but for people who don't have the information, it looks like a bug)
An option to revert back to the original colour scheme would be great.

But why change something that works well? 
I imagine there are good reasons (accessibility I guess). Does someone have an official link from Elementor that gives the reasons?

**Comment by taropaa** [comment]
Dark mode also makes the form fields background in the design black. Come on Elementor, really?

**Comment by fabigrafo** [comment]
I mean it's just in alpha stadium right now, but please let us change the color by custom or bring back the new container blue. This was beautiful and perfect to work with it.

**Comment by momo-fr** [comment]
In addition to the pink which is not really to my taste, the dark mode poses a problem in the text fields of the editor, we no longer see the selected/unselected text. You have to at least have a contrast in focus, right?

**Comment by tizum** [comment]
This new color wasn't cool, I didn't particularly like it and I tested it using dark and white colors, I returned Elementor to a previous version, it could have more options. The fonts are also too small and hard to see, the new UI is not good, it lacks UX as well.

**Comment by dbfischer** [comment]
I just copied the /plugins/elementor/assets/css folder from 3.11 to 3.12 because the new UI colors hurt my eyes. Now I have the proven colors back which were much better. It's inconsiderate of users to make the UI uncomfortable for no good reason. It leaves me stunned.

**Reply by shred787** [reply]
I'll give this a try ASAP!

**Reply by dbfischer** [reply]
I think this is only a short term solution because the CSS will change with new versions. But it helps me for now. ;-)

**Comment by awakekat** [comment]
Please bring back the old Widget Panel UI colors - White buttons, gray background and the Maroon at the top. Why? You are confusing your users. The now 3.12 Widget Panel UI color of white on white cannot be winning any accessibility awards. Pink for color blind people can be difficult, and white on white is not something I want to strain my eyes with several hours a day. Bring it back or give us the option to customize it the way we want to.

**Comment by bengpengyap** [comment]
The new color is fine with me. The old color is also fine with me.

**Comment by cswebstuff** [comment]
This says it all  LOL:  https://www.reddit.com/r/elementor/comments/1264qtw/omg_elementor_pink_purple_color_scheme/

**Comment by black-eye** [comment]
So, this feature request has become no. 1 in less than a week and the official FB poll has got quite bold outcome as well (https://www.facebook.com/groups/Elementors/posts/5779878528804552/):

![obrazek](https://user-images.githubusercontent.com/5634870/230071358-86875a5b-4b42-40da-a006-24a99b0d34f8.png)

Are there any conclusions from the Elementor Team regarding the new UI? And please, don't be mistaken - this is not "a war against the pink". There are more problematic issues there, including:

- Some parts are too contrasty (basically 0:100 ratio)
- On the other hand, other parts are almost unreadable, because of the low contrast (e.g. selected text)
- Because of too high contrast some parts are indistinguishable from others (e.g. control sections)
- It's really hard to distinguish filled in values from defaults
- The white left panel is visually blending with the edited page, if the page bg is white (which is quite common).

Most of the issues has been reported repeatedly here on GitHub or in FB group.

And please, don't tell us that we will understand the current changes, when the new UI fully arrives. If the changes cannot be understood now, leave publishing them when ready.

**Reply by tizum** [reply]
+1

**Comment by 79ho3ein** [comment]
I worked with light mode for 10 minutes, it really hurt my eyes. White, pink and black colors in light mode are very annoying for the eyes. Because there is a big difference between these three colors. In addition, there are two other cases.

1- Why is the top and bottom bar in the widget panel black in light mode? In this case, it is not clear whether the editor is in dark or light mode. The presence of black color in the top and bottom bars is the main reason for eye damage. (on mode)

![1](https://user-images.githubusercontent.com/77163248/230082262-f385e457-791c-4280-9488-e386f005578d.png)

2- Why should the new top bar be black in light mode? I remember Nicholas Zein told me in the beta version that the reason is that the top bar should be noticeable and prominent. Understandable, but I've never seen a dark color used in light UI.

Finally, I hope the right decisions are made for the new ui, so that we can use the great and exciting features of the future in the beautiful editor environment.

**Reply by mcrowell** [reply]
Great points!

**Comment by Michaelgimii** [comment]
A question has been bothering me for a long time, I hope one of my friends knows the answer...

**Is the new Elementor editor exactly that?**

![editor](https://user-images.githubusercontent.com/88034902/230084763-1121a590-289b-4165-8bd3-ea97272cefe1.png)

I mean, won't the display of widgets, customization settings (alignment option, switcher button, slider, etc.) change? Let me ask my question like this. **When the top bar is active, we are looking at the second version of the Elementor editor?**

Doesn't the ui of the navigation panel, fader panel, history, etc. change? Are the performance enhancements of the 2nd version editor written in react already palpable, or is it too early for that? In general, what percentage of the second version of the editor do we see when we activate the top bar? Both in terms of ux, ui and performance. thanks

**Comment by cswebstuff** [comment]
The new UI is going to cause a lot more user errors.

**@black-eye made some excellent points...**

- Some parts are too contrasty (basically 0:100 ratio)
- On the other hand, other parts are almost unreadable, because of the low contrast (e.g. selected text)
- Because of too high contrast some parts are indistinguishable from others (e.g. control sections)
- It's really hard to distinguish filled in values from defaults
- The white left panel is visually blending with the edited page, if the page bg is white (which is quite common).

_Elementor may want to bring in an outside UI design company to provide a non-biased corporate culture perspective._

**Comment by 79ho3ein** [comment]
This is what I expect from light mode. 

![light mode](https://user-images.githubusercontent.com/77163248/230115945-fde066ba-3508-4f85-9e83-c5ab79ad8fe6.png)

I only spent an hour on css changes. It can definitely be better. Especially in the details that friends mentioned.

**Reply by fabigrafo** [reply]
Thanks! That's what I expect from a company like Elementor. Elementor should have a close team of users, developers and those responsible.

**Reply by Michaelgimii** [reply]
fantastic!
Seeing this UI, I can't stand Elementor light mode anymore. I hope someone from Elementor sees it.

**Reply by fabigrafo** [reply]
> fantastic!
> Seeing this UI, I can't stand Elementor light mode anymore. I hope someone from Elementor sees it.

Tag @nicholaszein here so he can see it. He is my hope for a better accessibility of the dev team.

**Comment by SteinBC** [comment]
My answer from a support member:

![support-01](https://user-images.githubusercontent.com/129859602/230303612-8d37e057-0784-4e02-a2e8-3748e4ccaf7b.jpg)

**Reply by dbfischer** [reply]
I got this answer from support:

_Unfortunately, at this moment, it's not possible to change the look of the editor. You are only able to change the light and dark themes.
The Editor and other Elementor screens have been recolored and simplified. The new colors are better adapted to Dark and Light modes and are optimized for accessibility.
The new color scheme will not impact any functionalities of the product._

**Reply by SteinBC** [reply]
"better adapted to Dark and Light modes" is just a late 1 April joke

**Reply by fabigrafo** [reply]
The idea is good, only the implementation is not.. 😵‍💫

**Reply by awakekat** [reply]
Well with some ingenuity and css knowledge one can do just about anything. I installed the Code Snippets plugin and found a PHP override for the CSS and made this to my liking:
![Screenshot 2023-04-24 at 4 42 11 PM](https://user-images.githubusercontent.com/5148843/234112766-349e8f5d-43e9-4ef7-92ef-f6fc378b4265.jpg)

**Reply by rezaport** [reply]
hi there, please tell us how you did it ?

**Comment by createscape** [comment]
I agree with this and would like to see the UI colours changed, or give us an option to change them easily.   The high contrast black and white hurts my eyes and is very distracting, and I think my clients will probably object to the pink.   If this doesn't change I will be exploring different options for new projects.

**Comment by fabigrafo** [comment]
Any news here?

**Comment by arcst** [comment]
Working with Elementor has become pure stress. The backend colors either have no contrast to each other or too much. In dark mode: If you hover the text edit buttons or insert a link, it becomes light gray in light gray. So you have to guess when you want to change a link or which button you just activated. What a bad usability!

**Comment by Henzxon** [comment]
The dev version with purple works better. Unfortunately that's risky move to use dev version on production sites. Their brand also comes with other color combinations like dark blue and dark pink. Those color gives more contrast regardless the preferred color scheme.

The visual builder should be focused on UX not on Cosmetics.

**Comment by zevrix** [comment]
Adding my voice to this. When I bought Elementor several years ago I didn't subscribe to some kind of Barbie UI. Pink color is just not associated with powerful professional software - let alone it's simply not pleasant to use. There are also several other UI changes that make it inferior to earlier versions. Just give us an option to restore the NORMAL pre-3.12 Elementor UI color scheme.

**Comment by zevrix** [comment]
P.S. Fun fact: Nobody, not a single user ever asked to change the UI to pink.

**Reply by bengpengyap** [reply]
Yeah, I did not even ask for it but I actually learned to love it. Glad Elementor was proactive.
